Assets/Scripts/Modules/Stats/IStatModule.cs

```diff
@@ -5,4 +5,5 @@
 
 public interface IStatModule {
 
+	void AssignTitle ();
```

Comment:
Same deal here, this method should take both an IUnit and a IMapTile as arguments